### PR TITLE
[embedded-app] Remove StepZilla, use custom component

### PR DESCRIPTION
**Pull request by illiakovalenko**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

### CODE REVIEW

File: samples/sitecore-embedded-jss-app/src/app/components/Wizard.js [hunk 1]
```diff
@@ -14,6 +14,10 @@ class Wizard extends React.Component {
     this.onFormValueChange = this.onFormValueChange.bind(this);
   }
 
+  componentDidMount() {
+    console.log('Here should be wizard');
+  }
+
```
Comment (ambrauer) [inline samples/sitecore-embedded-jss-app/src/app/components/Wizard.js:20]: Remove this componentDidMount/console.log?

**Review by ambrauer — APPROVED** [review]
Fixed a couple found issues (see commits), see one other comment re: console.log. Also, looks like there will need to be doc updates to remove StepZilla references here: https://jss.sitecore.com/docs/techniques/mvc-integration/client-side-embedding

File: samples/sitecore-embedded-jss-app/src/app/components/Wizard.js (hunk 1 above)
Comment (illiakovalenko) [reply samples/sitecore-embedded-jss-app/src/app/components/Wizard.js:20]: @ambrauer  It's expected, we had it in StepZillaPatched

Comment (illiakovalenko) [reply samples/sitecore-embedded-jss-app/src/app/components/Wizard.js:20]: nvm, it's not expected, will remove

**Pull request merged**